linters/psscriptanalyzer/plugin.yaml

```diff
@@ -7,27 +7,37 @@ downloads:
           windows: windows
           macos: macos
         url: https://github.com/PowerShell/PSScriptAnalyzer/releases/download/${version}/PSScriptAnalyzer.${version}.nupkg
+  - name: converttosarif
+    downloads:
+      - os:
+          linux: linux
+          windows: windows
+          macos: macos
+        url: https://psg-prod-eastus.azureedge.net/packages/converttosarif.${version}.nupkg
 tools:
   definitions:
     - name: psscriptanalyzer
       download: psscriptanalyzer
       shims: [PSScriptAnalyzer.psd1]
       known_good_version: 1.21.0
+    - name: converttosarif
+      download: converttosarif
+      shims: [ConvertToSARIF.psd1]
+      known_good_version: 1.0.0
 lint:
   definitions:
     - name: psscriptanalyzer
       files: [powershell]
-      tools: [psscriptanalyzer, pwsh]
+      tools: [psscriptanalyzer, converttosarif, pwsh]
```

Comment:
As a minor convention change, please change this to:
```yaml
main_tool: psscriptanalyzer
tools: [converttosarif, pwsh]
```

This is just more explicit so that the version that is enabled applies to `psscriptanalyzer`

Reply:
Done. FYI, I think this doc needs updated to reflect that syntax: https://docs.trunk.io/check/configuration/custom-linters#additional-tool-dependencies